package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-sass",
-  "version": "5.0.0",
+  "version": "5.1.0",
```

Comment:
@saper @xzyfer should we just bump this to 6 and also drop Node 10 with #3094? Would mean maybe waiting till the end of the week for cutting the release

Reply:
I'd like to have more research about actual compilers and C++ standards we need. Hope we do not need to use different once for different node versions?

I think one node version out, one node version in is a fair trade, we could do 6.x this way.

Reply:
I dropped the conditionals entirely here, and the CI seems OK, but that's no guarantee 😄

Reply:
We should with the same settings as node does. Maybe they are inherited from the node gyp files, we should check this. Long time since I've had look into this...

The c++0x workaround was for the old gcc on MacOS X I think...

Reply:
I'm fine with bumping to 6 :)